ext/MitsubaLoader/CElementShapeOBJ.h

```diff
@@ -0,0 +1,50 @@
+﻿#ifndef __C_ELEMENT_SHAPE_OBJ_H_INCLUDED__
+#define __C_ELEMENT_SHAPE_OBJ_H_INCLUDED__
+
+#include "IElement.h"
```

Comment:
your include paths on extension headers are all wrong...

See RULE nr. 5 in the readme
https://github.com/buildaworldnet/IrrlichtBAW/tree/master/ext

Reply:
where this rule come from actually?

Reply:
the justification is that the extension should not fuck-up your build system because you didn't add its directory to the search path.

Imagine that your project includes extension files, but extension files include their peers with relative paths, you're gonna have a fuck-up when two extensions have the same public include filenames, unless full paths relative to repo's `./include/` are specified